compiler/ilgen/ByteCodeIteratorWithState.hpp

```diff
@@ -54,6 +54,7 @@ template<typename ByteCode,
         _tryCatchInfo(comp->allocator("IlGen"))
       {
       _printByteCodes = (comp->getOutFile() != NULL && comp->getOption(TR_TraceBC) && (comp->isOutermostMethod() || comp->getOption(TR_DebugInliner) || comp->trace(OMR::inlining)));
+      _cannotAttemptOSR = comp->getOption(TR_EnableOSR) && !comp->isPeekingMethod() && methodSym->cannotAttemptOSRDuring(comp->getCurrentInlinedSiteIndex(), comp);
```

Comment:
Yes, `cannotAttemptOSRDuring()` will always safely return false for a site index of -1, as it is only concerned with methods above it on the stack.